IanGRap/Squish
# Circle bomb model

Contributors: Ian

Goal: Add the offical model to the circle bomb block.

How to test:
- Check to see that the circle bomb block is under the hazards list of the block spawner prefab
- Check to see that the circle bomb model has read and write set to off with compression on.
- Play a few matches. The circle bomb should eventually show up as a hazard.

Notes:
I ran into an issue where the second round of a match would load but the scene creator wouldn't put the starting area together properly. I think this is a separate race condition / unity build error, but I am still scared.
